### PR TITLE
Disable checkJS survey

### PR DESCRIPTION
**Pull request by sandersn**
Leaves in place the infrastructure of #26455, but disables the actual sending of the event.

### CODE REVIEW

File: src/server/editorServices.ts [hunk 1]
```diff
@@ -1616,13 +1616,6 @@ namespace ts.server {
             if (this.seenSurveyProjects.has(project.projectName)) {
```
Comment (sheetalkamat) [inline src/server/editorServices.ts]: Should we probably remove these methods and maps as well ? (and  the even name for compatibility?)

Comment (sandersn) [reply src/server/editorServices.ts]: Well, we will need all the same infrastructure for future surveys. The question is how likely we are to need a survey soon.
I can definitely remove the entire thing if you think it's the right thing.

File: src/server/protocol.ts [hunk 2]
```diff
@@ -2493,18 +2493,6 @@ namespace ts.server.protocol {
         projectName: string;
     }
 
-    export type SurveyReadyEventName = "surveyReady";
```
Comment (sheetalkamat) [inline src/server/protocol.ts]: I am not sure if we need to keep this? For compatibility?

Comment (sandersn) [reply src/server/protocol.ts]: That implies everything in this file, right? 

It does indeed look like VS Code still depends on this. (I mentioned removing the survey to Matt last month but didn't get a firm commitment on it.)
https://github.com/Microsoft/vscode/blob/248aea7fd51a703a7ab94ae5a582c85c11fbff33/extensions/typescript-language-features/src/typescriptServiceClient.ts

**Pull request merged**